src/battle_controller_player.c

```diff
@@ -1889,6 +1889,19 @@ static void HandleChooseActionAfterDma3(u32 battler)
     {
         gBattle_BG0_X = 0;
         gBattle_BG0_Y = DISPLAY_HEIGHT;
+        if (gBattleStruct->aiDelayTimer != 0)
```

Comment:
Should probably exclude wild battles without AI

Reply:
Wild battle without AI have aiDelayTimer set to 0, so this check already does the job

Reply:
SetAiLogicDataForTurn sets the delayTimer before checking if there is any AI, so it would be non zero

Reply:
Oh yeah you're right. At first I put it in a difference place and got confused.

Reply:
Fixed